flocker/apiclient/_client.py

```diff
@@ -536,3 +556,17 @@ def to_nodes(result):
         request.addCallback(to_nodes)
 
         return request
+
+    def this_node_uuid(self):
+        era = check_output(["flocker-node-era"])
```

Comment:
I think it would surprise most callers to learn that this method blocks. AFAICT, it's the only such method on the class. Please change to use an asynchronous subprocess runner.